client/my-sites/checkout/composite-checkout/test/util/index.js

```diff
@@ -699,6 +699,23 @@ export function createTestReduxStore() {
 	} );
 }
 
+export function mockPayPalEndpoint( endpointResponse ) {
+	const endpoint = jest.fn();
+	endpoint.mockReturnValue( true );
+
+	nock( 'https://public-api.wordpress.com' )
+		.post( '/rest/v1.1/me/paypal-express-url', ( body ) => {
```

Comment:
Should this URL be updated to 1.2?

Reply:
Sorry, yeah, I am actually working on that as we speak. I should have removed this from the review column until it was ready...